File: app/src/main/java/com/grace/placessearch/data/model/Events.java
package com.grace.placessearch.data.model;

import java.util.ArrayList;
import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Events {

    @SerializedName("count")
    @Expose
    private long count;
    @SerializedName("summary")
    @Expose
    private String summary;
    @SerializedName("items")
    @Expose
    private List<Item> items = new ArrayList<>();

    public long getCount() {
        return count;
    }

    public void setCount(long count) {
        this.count = count;
    }

    public String getSummary() {
        return summary;
    }

    public void setSummary(String summary) {
        this.summary = summary;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }

}

File: app/src/main/kotlin/com/grace/placessearch/common/data/model/Events.kt
package com.grace.placessearch.common.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

class Events {

    @SerializedName("count")
    @Expose
    var count: Long = 0
    @SerializedName("summary")
    @Expose
    var summary: String? = null
    @SerializedName("items")
    @Expose
    var items: List<Item> = ArrayList()

}